src/Rating/Rating.js

```diff
@@ -0,0 +1,51 @@
+import './Star.scss'
+import React, {Component} from 'react'
+import selectedStar from "../images/selected-star.png";
+import emptyStar from "../images/empty-star.png";
+
+class Rating extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      currentValue: undefined
+    }
+  }
+
+  makeStars = (props, numberOfStars = props.userRating.rating) => {
+    let stars = [];
+    for (let i = 0; i < 5; i++) {
+      let starSource = emptyStar
+      if (i < numberOfStars) starSource = selectedStar
+      stars.push(
+        <div className="stars-box">
+          <img
+            // starValue={props.starValue}
+            className="star-image"
+            alt="empty star icon"
+            src={starSource}
+            id={i + 1}
+            onMouseEnter={() => this.setState({ currentValue: i + 1 })}
```

Comment:
Love that we went with the stars idea and using the mouse events to set the value.

Reply:
Me too but I don't know if I can handle going for half-stars after today